public/css/themes/No2TypeB.css

```diff
@@ -0,0 +1,57 @@
+@import url('https://fonts.googleapis.com/css?family=Bellefair');
+
+html {
+    background-image: url("https://i.imgur.com/UtkSjdK.jpg");
+}
+
+body {
+    background-color: rgba(0, 0, 0, .55);
+    opacity: 1;
+}
+
+nav {
+    background-color: rgb(0, 0, 0, .55);
+}
+
+main { 
+    font-family: 'Bellefair', serif;
+}
+
+h1, h2, h3 {
+    font-weight: 300;
+    font-family: 'Bellefair', serif;
+}
+
+a {
+    color: rgb(209, 75, 75);
+}
+
+#navLeft a {
+    color: rgb(255, 255, 255, .7);
+}
+
+#navRight a {
+    color: rgb(255, 255, 255, .7);
+    font-family: 'Bellefair', serif;
+    font-size: 18px;
+}
+
+#player #song, #player #artist {
+    color: rgb(255, 255, 255);
+}
+
+#status, #version {
+    color: rgb(255, 255, 255, .8);
+}
+
+#footer {
+    color: rgb(255, 255, 255, .6);
+}
+
+#frontRequest #searchButton {
+    background-color: grey;
+}
+
+input[type=range]::-webkit-slider-thumb {
+    background: rgb(240, 75, 75);
+}
```

Comment:
No trailing newline... Tsk tsk, @kenellorando

Reply:
Oof, great catch. Wouldn't want that in prod!